ReactWindows/ReactNative.Shared/Bridge/NativeModuleRegistry.cs

```diff
@@ -175,6 +175,8 @@ public void Invoke(IReactInstance reactInstance, int methodId, JArray parameters
 
             public void WriteModuleDescription(JsonWriter writer)
             {
+                if (Target == null) throw new NullReferenceException("Target");
+
```

Comment:
It is a read only auto-property, however, the constructor does not check whether the INativeModule argument is null... Adding an ArgumentNullException to the constructor seems like a better way to ensure this read only property is never null?